docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -348,6 +348,10 @@ relevant information about the rule. Defaults to an empty array.
 is converted from a third-party security solution. Automatically created when
 it is not provided.
 
+|setup |String |Instructions on rule prerequisites such as required
```

Comment:
It might help to connect the field with the setup guide that users see in the UI:
```suggestion
|setup |String |Populates the rule's setup guide with instructions on rule prerequisites such as required
```